frame/curve-amm/src/lib.rs

```diff
@@ -824,6 +769,73 @@ pub mod pallet {
 	}
 
 	impl<T: Config> Pallet<T> {
+		pub fn create_pool(
```

Comment:
need not to forget make it transactional in pallet

Reply:
Sure, `pallet::call` not implemented yet.